.github/workflows/build_documentation.yml

```diff
@@ -0,0 +1,52 @@
+name: build_documentation
+on: [push, pull_request, workflow_dispatch]
+permissions:
+    contents: write
+
+jobs:
+  build_jammy_humble:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    container:
+      image: ghcr.io/bdaiinstitute/spot_ros2_jammy_humble:main
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install package
+        run: |
+          source /opt/ros/$ROS_DISTRO/setup.bash
+          # pull ros_utilites and spot_wrapper
+          # git submodule init 
+          # git submodule update
+          pwd 
+          ls
+          apt-get update
+          apt install wget 
+          ./install_spot_ros2.sh
+          # go to root folder
+          cd ../../
+          colcon build --symlink-install
+          source install/local_setup.bash
```

Comment:
If you are building the package, might just think about moving the actual build docs steps to `test.yml`? This doesn't need to be a separate workflow and can be just a part of build, test, build docs / deploy.

Reply:
I have moved everything to `test.yml`